### PR TITLE
update linkspeed for wired networks to fail to N/A instead of crashing module

### PR DESCRIPTION
**Pull request by hsartoris**
#1211 
See commit comment. Less frustrating behavior on failure to query interface for link speed.

### CODE REVIEW

File: src/adapters/net.cpp [hunk 1]
```diff
@@ -306,6 +306,7 @@ namespace net {
    * about the current connection
    */
   string wired_network::linkspeed() const {
+    if (m_linkspeed == -1) return "N/A";
```
Comment (patrick96) [inline src/adapters/net.cpp]: We could probably just have the function only read:
```cpp
  string wired_network::linkspeed() const {
    return m_linkspeed == -1 ? "N/A" : (to_string(m_linkspeed) + " Mbit/s");
  }
```

Don't really know why 0 should print out `???`

**Comment by severinkaderli** [comment]
Is there any update on this PR? I also suffer from this problem and would like this fix.

File: src/adapters/net.cpp (hunk 1 above)
Comment (hsartoris) [reply src/adapters/net.cpp]: Finally remembered to make this change; see https://github.com/polybar/polybar/pull/1772/commits/9809d9e9c75c74f98f863931f949543ff1d3c4aa

**Comment by hsartoris** [comment]
I don't really know much about codecov, but I can't see why there would be decreased coverage on files that weren't changed. @patrick96 merge anyway?

**Comment by patrick96** [comment]
That sometimes happens with older PRs because the number of files tracked is different between this branch and `master` I think. Doesn't really matter.

Thanks a lot :)

**Comment by hsartoris** [comment]
No problem; thanks for all the work you do!

**Pull request merged**